### PR TITLE
monero: 0.14.1.2 -> 0.15.0.0

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change

Continuing the update in #73124. This PR takes care of monero-gui.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change
- [x] Tested execution of all binary files
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @ehmry

### CODE REVIEW

**Comment by rnhmjoj** [comment]
I tested the connection of both monero-wallet-cli and monero-gui to a local daemon. Everything seems to be working.

File: pkgs/applications/blockchains/monero-gui/default.nix [hunk 1]
```diff
@@ -1,27 +1,27 @@
-{ mkDerivation, lib, makeDesktopItem, fetchFromGitHub
+{ stdenv, wrapQtAppsHook, makeDesktopItem, fetchFromGitHub
```
Comment (ehmry) [inline pkgs/applications/blockchains/monero-gui/default.nix]: Is it necessary to pass `stdenv`? This package is imported with `libsForQt5.callPackage`, and its preferable to use the `mkDerivation` that it provides.

Comment (rnhmjoj) [reply pkgs/applications/blockchains/monero-gui/default.nix]: I know, however it seems `mkDerivation` will be no longer necessary/remove in the future... #71089.

Comment (ehmry) [reply pkgs/applications/blockchains/monero-gui/default.nix]: Ok, good to know, thanks.

**Review by ghost — APPROVED** [review]

**Pull request merged**